app/src/main.cpp

```diff
@@ -93,25 +149,42 @@ void loop() {
       Serial.println("Mode: OVERDUB");
     }
   }
+  */
 
   if (buttons.record.fallingEdge()) {
     Serial.println("Record Button Pressed");
     switch (status) {
     case Status::Record:
+      disp.SetRecordButton(false);
+      disp.SetStopButton(false);
+      disp.SetPlayButton(true);
+      disp.displayTrack("Playing");
```

Comment:
Mentioned earlier, this can all be accomplished with a single function call.

Reply:
fixed